src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -286,6 +286,11 @@ protected function get_views() {
 
 		foreach ( $origins as $origin => $count ) {
 			$origin_instance = Tribe__Events__Aggregator__Records::instance()->get_by_origin( $origin );
+
+			if ( null === $origin_instance ) {
+				continue;
```

Comment:
That's probably a really good idea. I'm still not sure why the origin was skipped. My best guess was a conflict overrode the record somehow.

I'll enhance it.